README.md

```diff
@@ -140,6 +140,14 @@ no_proxy: *.example.local,*.other.global,192.168.0.0/16
 no_proxy: long.domain.name.local,example.local
 ```
 
+## Changing default log level ##
+
+To lower `snyk-monitor`'s logging verbosity `log_level` value could be set to one of these options:
+* `'WARN'`
+* `'ERROR'`
+
+By default, `log_level` is `'INFO'`
```

Comment:
I'm really sorry about this - but can you please add a `.` in the end? :) to match the style of the docs